src/hub/permissions.py

```diff
@@ -8,6 +8,15 @@ class CreateHub(RuleBasedPermission):
     def satisfies_rule(self, request):
         return request.user.reputation >= 100
 
+class UpdateHub(RuleBasedPermission):
```

Comment:
I created this PR off of https://github.com/ResearchHub/researchhub-backend/pull/972 which will be merged for and this change will not appear